test/bun.js/websocket.test.js

```diff
@@ -131,3 +133,50 @@ describe("WebSocket", () => {
     gc(true);
   });
 });
+
+describe("websocket in subprocess", () => {
+  it("should exit", async () => {
+    const server = Bun.serve({
+      port: 8765,
+      fetch(req, server) {
+        if (server.upgrade(req)) {
+          return;
+        }
+
+        return new Response("http response");
+      },
+      websocket: {
+        open(ws) {
+          ws.send("hello websocket");
+        },
+        message(ws, message) {
+          expect(message).toEqual("hello");
```

Comment:
can you set a boolean that this function was called in the test

Reply:
updated